2-structured-data/test/e2e/CloudSqlTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Samples\Bookshelf;
+
+require_once('BookshelfTestBase.php');
```

Comment:
If the base class is moved to `test/lib`, it will be picked up by the autoloader (see [composer.json](https://github.com/GoogleCloudPlatform/getting-started-php/blob/e2e-published/2-structured-data/composer.json#L4)), and then we can remove this line.